src/notebook/notebookKernel.ts

```diff
@@ -163,9 +163,12 @@ export class JuliaKernel {
                 const execution = this._currentExecutionRequest
                 if (execution) {
                     execution.appendOutput(new vscode.NotebookCellOutput(items.map(item => {
-                        if (item.mimetype == 'image/png' || item.mimetype == 'image/jpeg') {
+                        if (item.mimetype === 'image/png' || item.mimetype === 'image/jpeg') {
                             return new vscode.NotebookCellOutputItem(Buffer.from(item.data, 'base64'), item.mimetype)
                         }
+                        else if (item.mimetype.endsWith('+json')) {
+                            return vscode.NotebookCellOutputItem.json(item.data, item.mimetype)
+                        }
```

Comment:
@DonJayamanne if I look at the saved version of a notebook file, this seems to properly set JSON style output. But for the vega-lite MIME types for example it doesn't render at all. But I think that might be a problem in the Jupyter extension at the moment, right?

Reply:
Let me check.

Reply:
I merged this for now, and then will test with the vega fixes in Jupyter.